controllers/gce/controller/controller.go

```diff
@@ -330,7 +329,7 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 	// Record any errors during sync and throw a single error at the end. This
 	// allows us to free up associated cloud resources ASAP.
 	var syncError error
-	if err := lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, nodePorts); err != nil {
+	if err := lbc.CloudClusterManager.Checkpoint(lbs, nodeNames, allNodePorts); err != nil {
```

Comment:
I'm OK with that assumption, but in the "no valid nodeports" case, it was flapping the IG.  I am open to alternate solutions :)